tools/build_defs/repo/http.bzl

```diff
@@ -88,6 +88,8 @@ _http_archive_attrs = {
     "build_file_content": attr.string(),
     "patches": attr.label_list(default=[]),
     "patch_tool": attr.string(default="patch"),
+    "patch_arguments": attr.string_list(default=["-p0"]),
+    "patch_verbose": attr.bool(default=False),
```

Comment:
As the utils are shared, the corresponding arguments also have to be added to `git.bzl` as well.